first-party/build.gradle

```diff
@@ -290,6 +293,13 @@ subprojects {
     }
   }
   /* TEST COVERAGE */
+
+  /* SONARQUBE */
+  sonarqube {
+    properties {
+    }
+  }
+  /* SONARQUBE */
```

Comment:
Looks empty. Does this actually do something? Is this required?

Reply:
Ah good question. This is required for ./gradlew to recognize the `sonarqube` task. The properties can either be specified through the build file or through the command line. I moved those properties to the build file since the initial setup looked a little strange.